authors/apps/articles/models.py

```diff
@@ -48,7 +52,50 @@ def _get_unique_slug(self):
             num += 1
         return unique_slug
 
+    def get_likes_or_dislkes(**kwargs):
+        """
+        Fetches the number of likes and dislikes
+
+        args:
+            Keyword arguments:
+            These include the model,
+            id of an article and
+            the like status of the article
+
+        Returns:
+            filtered_likes(int): number of likes or dislikes of an article
+        """
+        likes = kwargs.get('model').objects.all().filter(
+            like_article=kwargs.get('like_article')
+        )
+        filtered_likes = likes.filter(article_id=kwargs.get('article_id'))
```

Comment:
@Rhytah , you should perform a filter with both params to avoid making unnecessary queries